application/common/models/Password.php

```diff
@@ -48,7 +50,7 @@ public function rules(): array
                 'password', 'string',
             ],
             [
-                'password', 'checkRecentlyUsed',
+                'password', 'checkRecentlyUsed', 'on' => self::SCENARIO_DEFAULT
```

Comment:
Sorry, I am getting Yii 2.x upgrade triggered. You should probably figure out "new Password(['scenario' => 'update']);" or create instead of update. "on default" seems pointless?

Reply:
Not sure I understand your comment fully, but one of the suggestions on an earlier PR was to reuse the existing Password record so the metadata doesn't change.